app/javascript/hyacinth_v1/components/projects/publish_targets/PublishTargetForm.js

```diff
@@ -2,12 +2,14 @@ import React, { useState } from 'react';
 import { Form, Collapse } from 'react-bootstrap';
 import { useMutation } from '@apollo/react-hooks';
 import { useHistory } from 'react-router-dom';
+import { lowerCase } from 'lodash';
```

Comment:
Might be good to swap lodash for `"string".toLowerCase()` in this component too.